bin/rose-tutorial

```diff
@@ -30,8 +30,7 @@
 #     Make a copy of one of the tutorial SUITE in the cylc-run directory.
 #
 #-------------------------------------------------------------------------------
-# shellcheck source=lib/bash/rose_init
-. rose_init
+set -eu
```

Comment:
I think that this may be broken - 
```bash
$ rose tutorial
rose tutorial SUITE [DIR]

Available tutorial suites:
find: './etc/tutorial/': No such file or directory
```

Reply:
Currently works for me in an editable installation. probably more by chance than anything though.

I would like to punt this to an issue as the tutorials themselves are currently broken anyway, this would be better tackled as part of documentation work (will require a lot of docs updates etc):

https://github.com/metomi/rose/issues/2454

Reply:
I fixed it with
```diff
--- a/bin/rose-tutorial
+++ b/bin/rose-tutorial
@@ -37,7 +37,7 @@ usage () {
     echo 'rose tutorial SUITE [DIR]'
     echo
     echo 'Available tutorial suites:'
-    find "./etc/tutorial/" -maxdepth 1 -exec basename {} \; | sed 's/^/    /'
+    find "${ROSE_HOME}/etc/tutorial/" -maxdepth 1 -exec basename {} \; | sed 's/^/    /'
 }
 
 api-key () {
@@ -45,6 +45,8 @@ api-key () {
     sort -R "${ROSE_HOME}/etc/tutorial/api-keys" | head -n 1
 }
 
+ROSE_HOME=$(python3 -c "from pathlib import Path; import metomi.rose; print(Path(metomi.rose.__file__).parent.parent.parent)")
+
 if [[ $# == 1 ]]; then
     DIRECTORY="$HOME/cylc-run/$1"
 elif [[ $# == 2 ]]; then
@@ -87,7 +89,7 @@ fi
 echo -n "Copying tutorial files to ${DIRECTORY} ... "
 
 mkdir -p "$(dirname "${DIRECTORY}")"
-if run rsync -rLptgoD "${ROSE_HOME}/etc/tutorial/$1/" "${DIRECTORY}" \
+if rsync -rLptgoD "${ROSE_HOME}/etc/tutorial/$1/" "${DIRECTORY}" \
     --exclude='.validate'; then
     find "${DIRECTORY}" -type f \
         -exec sed -i "s/DATAPOINT_API_KEY/$(api-key)/" {} \;
```